module/VuFind/src/VuFind/View/Helper/Root/RecordLinker.php

```diff
@@ -304,7 +305,13 @@ public function getVersionsSearchUrl($driver)
         ];
 
         $urlHelper = $this->getView()->plugin('url');
-        return $urlHelper($route, [], ['query' => $urlParams]);
+        $url = $urlHelper($route, [], ['query' => $urlParams]);
+        if (strlen($url) > 2048) {
```

Comment:
Should we use mb_strlen? (Probably doesn't make much difference one way or the other in this instance... but I'm trying to be in the habit of using mb_strlen when appropriate).

Reply:
I don't think the URL can ever contain multibyte characters. Even international domain names should be encoded as IDNA that's ascii safe.

Reply:
Okay, in that case, I'll withdraw my suggestion. :-)